### PR TITLE
Render the other responses on the iopub channel

### PR DESCRIPTION
**Pull request by rgbkrk**
Note that this doesn't actually handle streams appropriately (first come first serve).

At the very least though, non display protocol outputs (typical stdout, stderr) get printed.

I've set the class of the stream output to `stream-${name}` (e.g. `stream-stdout`). No styling here at the moment.

### CODE REVIEW

**Comment by rgbkrk** [comment]
This does fairly lame rendering of errors (newlines aren't showing up), but you at least get a full traceback.

File: index.html [hunk 1]
```diff
@@ -9,12 +9,17 @@
...
-    
+
     <script>
-      require('ipc').on('display', function(message) {
+      var ipc = require('ipc');
+      ipc.on('display', function(message) {
+        document.getElementById('container').innerHTML = message;
+      });
+
+      ipc.on('stream', function(message) {
```
Comment (smashwilson) [inline index.html]: The "stream" and "display" handlers are identical, now.

File: lib/jupyter.js [hunk 2]
```diff
@@ -6,14 +6,15 @@ module.exports = {
...
 
 function formConnectionString(config, channel) {
   var portDelimiter = ":";
   if (config.transport !== "tcp") {
     portDelimiter = "-";
   }
-  
-  return config.transport + "://" + config.ip + portDelimiter + config[channel + "_port"]; 
+
+  return config.transport + "://" + config.ip + portDelimiter + config[channel + "_port"];
```
Comment (smashwilson) [inline lib/jupyter.js]: :fire: whitespace. Yesss

File: index.html (hunk 1 above)
Comment (rgbkrk) [inline index.html]: Well... yeah I just knew they'd be separate later.

**Pull request merged**